src/presentational-components/shared/RemoveModal.js

```diff
@@ -9,7 +9,7 @@ const RemoveModal = ({ title, text, onClose, onSubmit, isOpen, confirmButtonLabe
 
   return (
     <Modal
-      className="ins-c-rbac__dialog--warning"
+      className="rbac"
```

Comment:
Yes we need to use it actually. The apps are scoped in `insights.appname` class and all app styles are prefixed by this class. We are preventing global CSS conflicts between apps (if some app modifies pf-c-card for example). But modals are appended to the body, not to the wrapping root element so they are outside of the CSS scope. So we need to add the class manually. 

The class scoping is handled by chrome so I am thinking about adding the class to the body element instead. But we neet this as a hotfix.

Reply:
This is actually needed to make the styling work. Styles are scoped now to prevent the application changing CSS of another application. Modals and wizards are append to body - they are out of the scope - so we need to wrap them into element with classname which is named the same as insights.appname

Reply:
Ah okay that makes more sense. Works for me!